recipes/libtmux/conda-forge.yml

```
@@ -0,0 +1,3 @@
+noarch_platforms:
+  - linux_64
+  - win_64
```

My guess is that you want it on macOS but not Windows, right?

```suggestion
  - osx_64
```

---

I don't think we need `osx_64` as a platform since the `linux_64` and `osx_64` builds are the same (and are only constrained by `__unix`).
I would remove `win64` platform and selectors altogether but it seems in this CI the Windows builds will run nonetheless and fail, hence the need to make a dummy windows package.

---

I believe we do need it here when makin os-noarch, or the binary will be installable only on Linux.

---

My understanding of `noarch_platforms` is that to run mutually exclusive `noarch` packages, one may still need different  platforms, for instance if dependencies are different. However this is not the case of `macos` and `linux` here which should have the same `noarch` package, in the same way that a truly `noarch` package can be built on `linux-64` only.
What do you think? Am I missing something ?

Running a clean build locally with `run_locally.py linux64`,  `cat build_artifacts/linux-64/current_repodata.json`  is empty and `cat build_artifacts/noarch/current_repodata.json`  outputs
```json
{
  "info": {
    "subdir": "noarch"
  },
  "packages": {},
  "packages.conda": {
    "libtmux-0.36.0-pyhca7485f_0.conda": {
      "build": "pyhca7485f_0",
      "build_number": 0,
      "depends": [
        "python >=3.8",
        "tmux"
      ],
      "legacy_bz2_md5": null,
      "license": "MIT",
      "license_family": "MIT",
      "md5": "b7591b3a7a24a5e3ec81231865222012",
      "name": "libtmux",
      "noarch": "python",
      "sha256": "502e150ef97a3ee747a91b6338bb3be76cb535cd8fd6f78cb959a39686c71048",
      "size": 48595,
      "subdir": "noarch",
      "timestamp": 1712670081414,
      "version": "0.36.0"
    },
    "tmuxp-1.45.0-pyhca7485f_0.conda": {
      "build": "pyhca7485f_0",
      "build_number": 0,
      "depends": [
        "colorama >=0.3.9",
        "libtmux >=0.36",
        "python >=3.8",
        "pyyaml >=6.0"
      ],
      "legacy_bz2_md5": null,
      "license": "MIT",
      "license_family": "MIT",
      "md5": "b67b0c8dbc4c43c234294a21891ace9f",
      "name": "tmuxp",
      "noarch": "python",
      "sha256": "49f76bc22975ab9dc18f653c7e94d255fe80f81f963c9d7e8c64d9ae04195bfd",
      "size": 47418,
      "subdir": "noarch",
      "timestamp": 1712670121330,
      "version": "1.45.0"
    }
  },
  "removed": [],
  "repodata_version": 1
}
```

---

> What do you think? Am I missing something ?

Nope. I was probably missing that you did not had any specific dependency for macOS. Sorry for the red-herring.